src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -1,12 +1,62 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
 package frc.robot.subsystems;
-import edu.wpi.first.wpilibj.DigitalInput;
+
+import edu.wpi.first.wpilibj.AnalogInput;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
 
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonFX;
+import edu.wpi.first.wpilibj.motorcontrol.Talon;
 
+public class ArmSubsystem extends SubsystemBase {
+  /** Creates a new ArmSubsystem. */
+  private AnalogInput WristAnalogInput;
+  private AnalogInput ShoulderAnalogInput;
+  private WPI_TalonFX WristFalcon;
+  private WPI_TalonFX ShoulderFalcon;
 
-public class ArmSubsystem extends SubsystemBase{
-    
     public ArmSubsystem() {
-
+    ShoulderAnalogInput = new AnalogInput(0);
+    WristAnalogInput = new AnalogInput(0);
+    ShoulderFalcon = new WPI_TalonFX(Constants.shoulderFalconId);
+    WristFalcon = new WPI_TalonFX(Constants.wristFalconId);
+     this.setName("ArmSubsystem");
+    }
+    public void armMovePossition (){
+      //  for(int getArmPossition=1; 1<50;i++);
+      // need range of voltage/ range of motion of Arm
+      // need exact voltage value of 3 or 4 possitons
+      // some sort of if statement will be used to
+      // move the arm to a new possition
+    }
+    
+    public  void getArmPossition (){
```

Comment:
suggestion: getters will return data. So change this to
````suggestion
public double getArmPosition () {
````